gradle.properties

```diff
@@ -6,4 +6,4 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 publishPluginVersion = 0.9.7
-awsSdkS3Version = 1.11.73
+awsSdkS3Version = 1.11.700
```

Comment:
Please update the version to 1.11.783 which has Milan region support in region enumerator defined in Regions.java